Libraries/pbxspec/Headers/pbxspec/PBX/ProductType.h

```diff
@@ -65,8 +65,29 @@ class ProductType : public Specification {
         bool parse(plist::Dictionary const *dict);
     };
 
+public:
```

Comment:
Cool thanks.

Followup question: is there an external resource I can read that will help educate me on how xcodebuild builds things so I don't have to ask you as many stupid questions? :)